core/assets/jsons/global.json

```diff
@@ -254,8 +254,7 @@
   },
   "flarecount": {
     "texture": {
-      "active": "activeflarecounter",
-      "inactive": "inactiveflarecounter"
```

Comment:
Can you remove `inactiveflarecounter` from assets too so it isn't being loaded?

Reply:
It's already removed